pkg/kubelet/volumemanager/reconciler/reconciler_common.go

```diff
@@ -250,9 +250,10 @@ func (rc *reconciler) waitForVolumeAttach(volumeToMount cache.VolumeToMount) {
 		// Volume is not attached to node, kubelet attach is enabled, volume implements an attacher,
```

Comment:
Is it legacy behavior? Should we deprecate and remove it in a future release?

Reply:
Yes, it's very legacy. I doubt anyone uses this code.

Reply:
According to https://github.com/kubernetes/kubernetes/issues/20262#issue-129377066

Kubelet volume attach/detach enabled, this mode is primarily provided to support two scenarios:

- Support rolling upgrades of Kubernetes 
- To support volumes defined in static pods defined on standalone (no master) kubelets.

The following plugins implement the attach/detach interface:

- csi (don't support kubelet volume attach/detach)
- fc (attach is noop, attach logic is in WaitForAttach)
- flexvolume (deprecated but still available, flex may support both attach/detach in kubelet or controller)
- iscsi (attach is noop, attach logic is in WaitForAttach)

So we only take care of the flexvolume plugin. Once the flexvolume plugin is removed, the kubelet attach/detach feature can be removed as well.

The flag `--enable-controller-attach-detach` is not marked as deprecated. Can we mark it as deprecated in the next release? @jsafrane